docs/workflow/testing/mono/testing.md

```diff
@@ -6,6 +6,18 @@ The runtime tests will be available at a later date.
 ## Running Library Tests
 Running library tests against Mono is straightforward regardless of configuration.  Simply run the following commands:
 
+0. Build the "libraries" subset (see [building libraries workflow](../../building/libraries/README.md))
+Mac OS X (With Homebrew openSSL)
+```bash
+OPENSSL_ROOT_DIR="/usr/local/opt/openssl@1.1" ./build.sh --subsetCategory libraries
```

Comment:
`OPENSSL_ROOT_DIR` is not needed to build CoreCLR or libraries. Why does Mono need that?

Reply:
it's an alternative to https://github.com/dotnet/runtime/blob/master/docs/workflow/requirements/macos-requirements.md#openssl. I don't think we really want to mention this here.